python/tchannel/tornado/message_builder.py

```diff
@@ -0,0 +1,65 @@
+from ..messages import Types
+
+
+class MessageBuilder(object):
+
+    ARGS = ["arg_1", "arg_2", "arg_3"]
+
+    def __init__(self):
+        """key: message_id
+           value: incomplete messages
+        """
+        self.message_buffer = {}
+
+    @staticmethod
+    def _get_cur_pending_arg(message):
+        cur_arg = 0
+        if message.arg_3 != "":
```

Comment:
Do

    if not message.arg_3:

That will handle both, None and empty string cases.